infra/terraform/testground.tf

```diff
@@ -66,35 +66,44 @@ resource "aws_security_group" "default" {
   }
 }
 
+data "aws_ami" "base" {
+  most_recent = true
+  owners      = ["self"]
+  filter {
+    name   = "tag:Name"
+    values = [var.aws_ami]
+  }
+}
+
 resource "aws_instance" "testground_manager" {
-  ami                    = "${lookup(var.aws_amis, var.aws_region)}"
-  availability_zone      = "${var.aws_availability_zone}"
-  instance_type          = "${var.aws_instance_type_manager}"
-  key_name               = "${var.key_name}"
+  ami                    = data.aws_ami.base.id
+  availability_zone      = var.aws_availability_zone
+  instance_type          = var.aws_instance_type_manager
+  key_name               = var.key_name
   vpc_security_group_ids = ["${aws_security_group.default.id}"]
   iam_instance_profile   = "EC2-Full-API-Access-Dangerous"
-  subnet_id              = "${aws_subnet.default.id}"
+  subnet_id              = aws_subnet.default.id
   private_ip             = "172.16.0.10"
 
   tags = {
-    TG     = "${var.tag}"
+    TG     = var.tag
     Name   = "${var.tag}-manager"
     TGRole = "manager"
   }
 }
 
 resource "aws_instance" "testground_redis" {
-  ami                    = "${lookup(var.aws_amis, var.aws_region)}"
-  availability_zone      = "${var.aws_availability_zone}"
-  instance_type          = "${var.aws_instance_type_redis}"
-  key_name               = "${var.key_name}"
+  ami                    = data.aws_ami.base.id
+  availability_zone      = var.aws_availability_zone
+  instance_type          = var.aws_instance_type_redis
+  key_name               = var.key_name
```

Comment:
AFAIK it doesn't matter which key we use to create VMs (so using personal keys such as `jim-us-west-2` is fine), as long as we then replace the `authorized_keys` file. I think it would be a good practice to collect the keys from team members and add everyone's key to the `authorized_keys` right away as a first step when configuring masters and workers.

Reply:
That is a feature I'm hoping to implement as part of #186 - agree that we could have a default that matches a keypair we have on our AWS account. Non-PL users would have to supply their own, of course.